components/turret.py

```diff
@@ -267,6 +273,19 @@ def _motor_is_finished(self) -> bool:
             < self.ACCEPTABLE_ERROR_SPEED
         )
 
+    def _turnaround_sufficient(self) -> bool:
+        if self.turnaround_direction == self.POSITIVE:
+            if self.motor.getSelectedSensorPosition() > (
+                self.current_target_counts - self.PI_OVER_2_IN_COUNTS
+            ):
+                return True
+        else:
+            if self.motor.getSelectedSensorPosition() < (
+                self.current_target_counts + self.PI_OVER_2_IN_COUNTS
+            ):
+                return True
+        return False
```

Comment:
Might be less readable but you could also do it like this, up to you.
```suggestion
        sign = (self.turnaround_direction == self.POSITIVE) * 2 - 1
        if self.motor.getSelectedSensorPosition() > (
            self.current_target_counts - sign * self.PI_OVER_2_IN_COUNTS
        ):
            return True
        else:
             return False
```